package.json

```diff
@@ -95,7 +95,7 @@
     ]
   },
   "dependencies": {
-    "@typescript-eslint/utils": "^5.10.0"
+    "@typescript-eslint/utils": "^6.0.0"
```

Comment:
<img width="359" alt="image" src="https://github.com/jest-community/eslint-plugin-jest/assets/1404810/6c7bd8cc-7f81-44f8-8c67-cd80ac9b1420">

as well?

Reply:
we're still supporting `@typescript-eslint/eslint-plugin` v5 and don't use a lot of the actual rules so I don't think there's a strong advantage - I'll double check that but I don't think it should block landing this right now

Reply:
I think we can drop it - no reason too keep support when bumping version (when we're landing as major regardless)

Reply:
ok but can we do that in a PR after this so it gets a dedicated changelog entry?

Reply:
i.e. I think it makes sense to add support for further majors as a minor, but if we're doing a major release regardless, I don't think it's super valuable

Reply:
for sure!

Reply:
(sorry, I had the tab open and GH isn't super good with live updates 😅 let's blame rails!)

Reply:
Suggestion: update this to the last v6 version.
That way you're enforcing that package managers always give you the latest version, rather than trying to resolve to some random one with fewer bug fixes.

Reply:
I tend to favor not doing that to enable better deduplication downstream, given we consider this a very stable dependency